os/os.proto

```diff
@@ -135,19 +153,26 @@ message TransferRequest {
   // the Target already has the OS package version it will reply with
   // InstallResponse->Validated. In the case that the target is a
   // single Supervisor device, or the partner Supervisor does not have the OS
-  // image specified, it will respond with InstallResponse->TransferReady. In
-  // this case, the client MUST subsequently transfer the image. In the case
+  // image specified, it will respond with InstallResponse->TransferReady. 
+  // In this case, the client MUST subsequently transfer the image. In the case
   // that the image is available on the peer Supervisor of a dual Supervisor
   // system, it will respond with InstallResponse->SyncProgress. In this,
   // latter, case - the client does not need to transfer the OS image. This
   // value can also be set empty, in which case the OS package is forced
   // transferred to the Target. The Target MUST never validate that this value
   // matches the one in the InstallResponse->Validated message, that is the
   // Client's responsibility.
+  // Systems which install the image on all Supervisors in one command
+  // will either have the image on both Supervisors or neither Supervisor.
+  // Such systems do not need to sync the image from peer Supervisor and
+  // will respond with TransferReady. 
```

Comment:
A dual Supervisor Target which requires installing the entire system with one Install RPC, will either have the image on both Supervisors or neither Supervisor. Such Targets do not need to sync the image from peer Supervisor and MUST immediately respond with TransferReady